grpc-native/src/main/java/org/ballerinalang/net/grpc/Message.java

```diff
@@ -130,9 +138,15 @@ public Message(
         }
 
         BMap<BString, Object> bBMap = null;
+        BArray bArray = null;
         if (type.getTag() == TypeTags.RECORD_TYPE_TAG) {
             bBMap = ValueCreator.createRecordValue(type.getPackage(), type.getName());
             bMessage = bBMap;
+        } else if (messageName.equals("Timestamp")) {
+            TupleType tupleType = TypeCreator.createTupleType(
```

Comment:
Shall we have a constant for this?. It is unnecessary to create it as a variable for each and every call.